src/passkit.cs

```diff
@@ -2096,4 +2194,456 @@ interface PKStoredValuePassProperties
+	{
+		[Abstract]
+		[Export ("elements")]
+		PKIdentityElement[] Elements { get; }
+
+		[Abstract]
+		[Export ("intentToStoreForElement:")]
+		[return: NullAllowed]
+		PKIdentityIntentToStore IntentToStore (PKIdentityElement element);
+
+		[Abstract]
+		[Export ("addElements:withIntentToStore:")]
+		void AddElements (PKIdentityElement[] elements, PKIdentityIntentToStore intentToStore);
+	}
+
+	interface IPKShareSecureElementPassViewControllerDelegate {};
+
+	[iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV, NoMac]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface PKShareSecureElementPassViewControllerDelegate
+	{
+		[Abstract]
+		[Export ("shareSecureElementPassViewController:didFinishWithResult:")]
+		void DidFinish (PKShareSecureElementPassViewController controller, PKShareSecureElementPassResult result);
+
+		[Export ("shareSecureElementPassViewController:didCreateShareURL:activationCode:")]
+		void DidCreateShareUrl (PKShareSecureElementPassViewController controller, [NullAllowed] NSUrl universalShareUrl, [NullAllowed] string activationCode);
+	}
+
+	interface IPKVehicleConnectionDelegate {}
+
+	[iOS (16,0), Mac (13,0), Watch (9,0), NoTV, MacCatalyst (16,0)]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface PKVehicleConnectionDelegate
+	{
+		[Abstract]
+		[Export ("sessionDidChangeConnectionState:")]
+		void SessionDidChangeConnectionState (PKVehicleConnectionSessionConnectionState newState);
+
+		[Abstract]
+		[Export ("sessionDidReceiveData:")]
+		void SessionDidReceiveData (NSData data);
+	}
+
+	[NoWatch, iOS (16,0), Mac (13,0), MacCatalyst (16,0), NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKAutomaticReloadPaymentRequest // : NSCoding, NSCopying, NSSecureCoding // https://feedbackassistant.apple.com/feedback/11018799
+	{
+		[Export ("paymentDescription")]
+		string PaymentDescription { get; set; }
+
+		[Export ("automaticReloadBilling", ArgumentSemantic.Strong)]
+		PKAutomaticReloadPaymentSummaryItem AutomaticReloadBilling { get; set; }
+
+		[NullAllowed, Export ("billingAgreement")]
+		string BillingAgreement { get; set; }
+
+		[Export ("managementURL", ArgumentSemantic.Strong)]
+		NSUrl ManagementUrl { get; set; }
+
+		[NullAllowed, Export ("tokenNotificationURL", ArgumentSemantic.Strong)]
+		NSUrl TokenNotificationUrl { get; set; }
+
+		[Export ("initWithPaymentDescription:automaticReloadBilling:managementURL:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (string paymentDescription, PKAutomaticReloadPaymentSummaryItem automaticReloadBilling, NSUrl managementUrl);
+	}
+
+	[NoWatch, iOS (16,0), Mac (13,0), MacCatalyst (16,0), NoTV]
+	[BaseType (typeof (PKPaymentSummaryItem))]
+	interface PKAutomaticReloadPaymentSummaryItem // : NSCoding, NSCopying, NSSecureCoding // https://feedbackassistant.apple.com/feedback/11018799
```

Comment:
```suggestion
	[DisableDefaultCtor]
	interface PKAutomaticReloadPaymentSummaryItem // : NSCoding, NSCopying, NSSecureCoding // https://feedbackassistant.apple.com/feedback/11018799
```

Reply:
- here

Reply:
Removing this disable after discussing with Alex since it appears to be valid!